src/main/java/com/artipie/management/api/artifactory/GetPermissionSlice.java

```diff
@@ -56,23 +56,23 @@
 public final class GetPermissionSlice implements Slice {
 
     /**
-     * Artipie settings.
+     * Repository permissions.
      */
-    private final Storage storage;
+    private final RepoPermissions permissions;
```

Comment:
@genryxy please, replace ; with . at the end of the sentence

Reply:
@olenagerasimova this line is not comment. Probably the highlighted text is a bit of misleading in this case.